addOns/reports/src/main/java/org/zaproxy/addon/reports/ReportHelper.java

```diff
@@ -138,6 +138,18 @@ public static boolean hasSiteStats(String site, String prefix) {
         return !getSiteStats(site, prefix).isEmpty();
     }
 
+    public static Map<String, Long> getGlobalStats(String prefix) {
+        ExtensionStats extStats =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionStats.class);
+        if (extStats != null) {
+            InMemoryStats stats = extStats.getInMemoryStats();
+            if (stats != null) {
+                return stats.getStats(prefix);
+            }
+        }
+        return Collections.emptyMap();
+    }
+
```

Comment:
Does this also need a `hasGlobalStats` like the site stats methods above?

Reply:
You'll probably tell me it'd be there if it was needed, but I'm just asking so that you have a second thought. I'm fine if it isn't needed :grinning:

Reply:
Yeah, we dont need it yet, so no point in adding it?